package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
I wouldn't be that strict when it comes to the hex color format, but I'm happy to hear other opinions as well.